dubbo-remoting/dubbo-remoting-zookeeper-api/src/main/java/org/apache/dubbo/remoting/zookeeper/ZookeeperTransporter.java

```diff
@@ -27,16 +27,14 @@ public interface ZookeeperTransporter {
 
     String CURATOR_5 = "curator5";
 
-    String CURATOR = "curator";
-
     ZookeeperClient connect(URL url);
 
     void destroy();
 
     static ZookeeperTransporter getExtension(ApplicationModel applicationModel) {
         ExtensionLoader<ZookeeperTransporter> extensionLoader =
                 applicationModel.getExtensionLoader(ZookeeperTransporter.class);
-        return isHighVersionCurator() ? extensionLoader.getExtension(CURATOR_5) : extensionLoader.getExtension(CURATOR);
+        return extensionLoader.getExtension(CURATOR_5);
```

Comment:
The existence of ZookeeperTransporter is not very meaningful, so consider removing it.

Reply:
OK, but I think I will use another PR and issue to handle this task. Because the test cases are still growing, I want to finish this PR earlier. Is that OK?

Reply:
Please take another look. Do you think my code is correct? I removed the SPI and moved the ZookeeperTransporter module in order to create an instance of Curator5ZookeeperTransporter. @CrazyHZM Thanks for your review.

Reply:
@Stellar1999 ZookeeperTransporter should be in the API module, and it should be renamed ZookeeperClientManager or something else.

Reply:
Please take another look. Thanks for your guidance. @CrazyHZM 
If it's alright, I will fix the sample test case.